data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/events/VoluntaryExitEvent.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beaconrestapi.handlers.v1.events;
+
+import tech.pegasys.teku.spec.datastructures.operations.SignedVoluntaryExit;
+
+public class VoluntaryExitEvent extends Event<SignedVoluntaryExit> {
+
+  VoluntaryExitEvent(final SignedVoluntaryExit exit) {
+    super(SignedVoluntaryExit.SSZ_SCHEMA.getJsonTypeDefinition(), exit);
+  }
+}
```

Comment:
is this redundant? can we just use the SignedVoluntaryExit directly?

Reply:
should be pretty much the same as attestation and contributionAndProof, they're all the actual objects...